src/Tools.md

```diff
@@ -107,7 +107,7 @@ This should result in a rather lengthy output.
 When it finally finishes, each of the subdirectories of the `build` directory will contain binary files that you can load onto the robot.
 
 Connect the robot to the USB port of your host computer.
-Before flashing the binary to your RP2040 based board, the board must be placed into `BOOTSEL` mode.  On the Pololu 3Pi+ robot, hold the `B` button and press `RESET`.
+Before flashing the binary to your RP2040 based board, the board must be placed into `BOOTSEL` mode.  On the Pololu 3Pi+ robot, hold the `B` button, press `RESET`, then release the `B` button.
```

Comment:
What I wanted to express here was that users have to first release the `RESET` button then the `B` button to enter `BOOTSEL` mode; otherwise it will just reset the robot. Does this make sense?

Reply:
Edward - the instructions to hold the 'B' button, press 'RESET', and then release both buttons can actually be incorrect if 'B' is released slightly before 'RESET'. What Pei-Wei is suggesting is to emphasize this. I think that the second part should be "press and release 'RESET'" though, which seems clearer to me.